### DISCUSSION TITLE
Where is the "data connection toolkit"?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#10686] [Archive]
The [Directus docs](https://docs.directus.io/getting-started/introduction/) says:
_For developers, a data connection toolkit provides a dynamic REST+GraphQL API, JavaScript SDK, and options for authorization, caching, and more._
Is the toolkit installed all together with Directus by `npm init directus-project myproject` or some other way?

**Comment by rijkvanzanten** [comment]
Yep! That's a term for all the APIs we offer combined. There's no standalone "toolkit" to instal; the Platform _is_ the toolkit 🙂
